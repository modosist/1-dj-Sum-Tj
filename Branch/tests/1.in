1366 130539
918 110455
698 89116
1323 124247
155 128106
1565 105037
1993 141841
827 114479
1192 104635
1707 96187
836 133242
302 138544
1164 103204
1218 141135
1255 108218
1531 133009
1149 105740
759 96464
690 108935
1780 110953
1098 121413
1409 108240
577 106287
198 93528
594 141419
1650 117905
125 134316
1511 120385
787 118048
1060 87502
741 135258
94 95754
1806 139642
946 143516
1161 125071
732 119382
1519 109612
823 94314
285 101187
447 134577
716 98409
1692 116260
1455 139263
1303 97625
1427 140911
215 107263
1392 113928
1384 108840
65 108008
1373 103304
1851 96852
129 109791
942 92871
1313 115482
74 126757
1414 108309
942 90521
1826 140910
941 87108
1 116393
1803 119137
1033 89569
447 118921
848 109137
5 117022
83 137276
1074 93757
38 126783
768 143489
673 115123
957 131418
1907 99077
1398 142857
168 118850
740 96892
1388 93715
828 107991
1037 94154
1504 112535
611 97053
1523 99627
1407 110009
887 128527
1928 143555
1591 90646
170 118716
1028 118000
1217 89693
871 139403
442 89832
511 122436
1845 133395
83 111578
1104 125455
185 122088
1015 137401
809 114246
1474 104354
372 130977
436 113010
1968 123369
326 125445
960 105944
996 104321
1656 100132
921 120994
1125 131509
1409 102363
545 120262
338 92461
894 142967
1391 102960
1125 124436
1668 98410
1196 120174
1863 142775
145 93562
428 88827
311 121242
724 113572
1373 103557
1473 138487
1367 141168
1294 97959
93 131269
528 100206
1553 127306
735 121403
492 125941
648 115161
1224 105602
1032 113517
1377 105311
456 99410
1141 124173
361 126197
265 103653
1584 91002
637 138242
1579 88157
1692 122707
1768 117946
624 117551
970 138588
1556 95876
1865 140135
1140 112519
787 96670
103 94641
1789 127609
161 124938
1737 127743
245 99823
340 113165
1742 142724
261 104295
1853 102221
876 108107
1636 94129
616 113004
1439 124507
1643 132422
760 141646
355 134033
1547 102020
1792 96496
177 105332
470 102203
460 105295
643 131822
978 102920
824 89652
1 114640
934 99243
1320 128939
1356 91691
1021 137204
1980 102697
1817 95344
895 135492
1381 95485
54 133478
809 98780
148 125771
1662 90665
1300 143439
1609 132231
1346 130415
1336 128562
602 120185
1418 117996
1278 110376
710 142795
1180 125908
817 125012
1710 126320
266 117250
1572 106756
1880 125060
726 133730
1422 106770
1028 116261
613 88740
721 128643
552 142673
947 113178
1331 108517
1883 124267
1704 101799
1553 115984
1640 138294
74 132599
680 117870
189 111530
364 123926
297 112049
472 107904
8 141716
907 129549
1555 108003
143 97838
1064 118763
1092 107500
1049 95938
411 88341
1379 94712
490 134993
913 92311
945 131600
1444 127650
1250 110625
1486 138463
1579 114561
1118 118123
216 106056
592 107190
851 124846
691 101241
1104 133009
240 129819
691 122935
917 129175
1535 121928
839 107389
749 140330
76 89249
727 115450
974 139035
1349 122328
821 97069
782 117644
1580 93523
528 86576
1651 137404
540 90740
55 101367
152 102893
1470 113343
150 139639
1842 136884
1167 102125
247 109189
39 143854
1082 111097
469 111653
728 117901
1574 130419
1591 114739
1349 97291
391 100983
15 120950
1502 91936
1410 116874
1217 107219
1064 91607
835 141220
246 110444
517 108329
486 95008
754 129218
1182 128303
899 103434
1191 140710
17 108736
33 94126
1808 107200
768 91876
1270 92622
280 94577
1270 139464
832 125636
1216 86569
468 107385
1177 96184
553 126145
437 125827
685 102569
1610 87638
1458 101694
726 124784
0 0
