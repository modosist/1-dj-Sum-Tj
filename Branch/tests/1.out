opt:  8184364
10tk: 8270881